mlir/lib/Target/LLVMIR/ConvertFromLLVMIR.cpp

```diff
@@ -30,6 +30,11 @@ void registerFromLLVMIRTranslation() {
       llvm::cl::desc("Emit expensive warnings during LLVM IR import "
                      "(discouraged: testing only!)"),
       llvm::cl::init(false));
+  static llvm::cl::opt<bool> importEmptyDICompositeTypes(
+      "import-empty-di-composite-types",
```

Comment:
We could also call this something like "avoid-recursive-type-import" or something. I'm open to suggestions 😄

Reply:
Maybe `drop-di-composite-type-elements` or `strip-di-composite-type-elements` is a bit more precise? The current naming works as well.

Reply:
Changing to drop-di-composite-type-elements